plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -172,8 +175,8 @@ def test_configuration_set_via_cmd_and_default_config(hydra_sweep_runner) -> Non
             "hydra.sweeper.ax_config.max_trials=2",
             "hydra.sweeper.ax_config.early_stop.max_epochs_without_improvement=2",
             "quadratic=basic",
-            "quadratic.x=-5:-2",
-            "quadratic.y=-1:1",
+            "quadratic.x=interval(-5, -2)",
```

Comment:
how come in all the other tests you had to cast the range to int and in this one you don't?

Reply:
We are just ensuring that the config is properly set.